app/src/main/java/com/imdbmovieapp/data/remote/dto/popularMoviesDto/MoviesResultDto.kt

```diff
@@ -0,0 +1,15 @@
+package com.imdbmovieapp.data.remote.dto.popularMoviesDto
+
+data class MoviesResultDto(
+    val id: Int,
+    @SerializedName("genre_ids")
+    @Json(name = "genre_ids")
+    val genreIds: List<Int>,
+    @SerializedName("poster_path")
+    @Json(name = "poster_path")
+    val posterPath: String,
+    @SerializedName("release_date")
+    @Json(name = "release_date")
+    val releaseDate: String,
+    val title: String
+)
```

Comment:
This is not correct. As Bachana wrote in his comment, you should use @SerializedName if you are using Gson library, but if you are using Moshi library, you should use @Json

Reply:
As I've checked your Gradle, you are using Gson. Therefore remove @Json(name = " ") from every data class

Reply:
ok, understand now

Reply:
so i just have to leave for example:  @SerializedName("poster_path") and remove @Json(name = "poster_path")

Reply:
yes